nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/operations/DeleteFromcoldstorage.java

```diff
@@ -0,0 +1,57 @@
+package org.nuxeo.coldstorage.operations;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.nuxeo.coldstorage.helpers.ColdStorageHelper;
+import org.nuxeo.ecm.automation.core.Constants;
+import org.nuxeo.ecm.automation.core.annotations.Context;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.NuxeoException;
+import org.nuxeo.ecm.core.api.security.SecurityConstants;
+
+import java.util.Optional;
+
+import static javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR;
+
+/**
+ * Delete the main content associated with the input {@link DocumentModel} from cold storage.
```

Comment:
`Delete` -> `Deletes`
"from cold storage" is isolated at the end so it's not clear if it relates to "the main content" or "the input DocumentModel".
-> Rephrase, for instance: `Deletes from cold storage the main content associated with the input {@link DocumentModel}`